client/my-sites/upgrades/checkout/secure-payment-form.jsx

```diff
@@ -74,6 +78,17 @@ var SecurePaymentForm = React.createClass( {
 	},
 
 	render: function() {
+		if ( this.state.visiblePaymentBox === null ) {
+			return (
+				<EmptyContent
+					illustration='/calypso/images/drake/drake-500.svg'
+					title={ this.translate( 'Payments are not available' ) }
```

Comment:
I'm certainly not the best person to make this point but I can stop wondering if we shouldn't use a different title. `Payments are not available` sounds strange to my ears: what about `Purchases are not available` or `Checkout is not available` instead?